gsl_projet/models.py

```diff
@@ -39,11 +39,11 @@ def for_perimetre(self, perimetre: Perimetre | None):
         if perimetre is None:
             return self
         if perimetre.arrondissement:
-            return self.filter(demandeur__arrondissement=perimetre.arrondissement)
+            return self.filter(perimetre__arrondissement=perimetre.arrondissement)
         if perimetre.departement:
-            return self.filter(demandeur__departement=perimetre.departement)
+            return self.filter(perimetre__departement=perimetre.departement)
         if perimetre.region:
-            return self.filter(demandeur__departement__region=perimetre.region)
+            return self.filter(perimetre__departement__region=perimetre.region)
```

Comment:
Puisque c'est dénormalisé sur le périmètre, on peut faire ceci
```suggestion
            return self.filter(perimetre__region=perimetre.region)
```